components/pedigree/resources/src/main/resources/pedigree/view/templateSelector.js

```diff
@@ -45,7 +45,9 @@ var TemplateSelector = Class.create( {
             pictureBox.update("Loading...");
             this.mainDiv.insert(pictureBox);
             var href = getSelectorFromXML(objects[i], "link", "rel", "http://www.xwiki.org/rel/properties").getAttribute("href");
-            new Ajax.Request(href, {
+            //should use only path when running PhenoTips behind a reverse proxy via mod_proxy_http
+            var path = href.substring(href.indexOf("/rest"));
```

Comment:
This discards the appname from the path. You should substring from the first `/` after `//`:

    var path = href.substring(href.indexOf("/", href.indexOf("//") + 2);